doc/ExpertOptions.md

```diff
@@ -1067,6 +1067,20 @@ task concat {
   }
 }
 ```
+### Unsupported overrides
+Current version of dxCompiler **does not** support instance override for native app(let)s in the following cases:
+1. Specifications of system requirements (RAM/CPU/storage/etc.) is not supported. Use instance names instead e.g. `mem1_ssd1_v2_x4` (instance names may be not transferable between regions)
```

Comment:
IMHO, we can remove "(instance names may be not transferable between regions)", or add it in the section above, where we first refer to the naming. Otherwise it may be confusing why we mention regions here.

Reply:
fixed in [15cc191](https://github.com/dnanexus/dxCompiler/pull/333/commits/15cc191bb8ec1b4c6fed9c4abe0963b3291562ce), added reference to Global Workflow Recommendations